modules/motions/Rotation.cpp

```diff
@@ -20,17 +19,16 @@ void Rotation::run()
   MotorController& motorController = robot.getMotorControllerInstance();
   spikeapi::Clock& clock = robot.getClockInstance();
 
-  prepare();
   if(!isMetPreCondition()) return;
-
-  // 回転速度（mm/秒）で指定しモーターを制御
-  motorController.setLeftMotorSpeed(speed * leftSign);
-  motorController.setRightMotorSpeed(speed * rightSign);
+  prepare();
 
   while(isMetContinuationCondition()) {
-    // 回頭継続条件が満たされるまで待機（モーターが走行中）
+    // 動的モーター制御
+    updateMotorControl();
   }
 
   // モーターを停止
   motorController.stopWheelsMotor();
+  motorController.setLeftMotorPower(0);
+  motorController.setRightMotorPower(0);
```

Comment:
実は、MotorController には、両輪のパワー値を0にする便利な関数が定義されていたりする...
まぁ、パワー値を明示的に0にしている感は薄れてしまうから使わなくてもいいけどねー

```suggestion
  motorController.resetWheelsMotorPower();
```